apps/remix-dapp/src/locales/en/udapp.json

```diff
@@ -49,9 +49,14 @@
   "udapp.enterAMessageToSign": "Enter a message to sign",
   "udapp.hash": "hash",
   "udapp.signature": "signature",
-  "udapp.saveVmStateTitle": "Save VM state",
-  "udapp.saveVmStateLabel": "State Name",
-  "udapp.saveVmStateTip": "Saved VM states can be pinned as environment using Environment Explorer",
+  "udapp.forkStateTitle": "Fork VM state",
+  "udapp.forkStateLabel": "State Name",
+  "udapp.fork": "Fork",
+  "udapp.deleteVmStateTitle": "Delete VM state",
+  "udapp.deleteVmStateDesc1": "Deleting state of the selected environment will delete all your previously saved transaction details for current workspace.",
+  "udapp.deleteVmStateDesc2": "It will also delete the data for contracts deployed and pinned for this workspace.",
```

Comment:
It will also delete the data of contracts deployed and pinned to this VM in this Workspace.

Reply:
This is in continuation of first statement so I think We don't need to add `this VM` again

Reply:
Then just change the text to this:
It will also delete the data of contracts deployed and pinned in this Workspace.

Reply:
updated